python/whylogs/viz/notebook_profile_viz.py

```diff
@@ -213,6 +214,18 @@ def set_profiles(
         self._target_view = target_profile_view
         self._ref_view = reference_profile_view
 
+    def profile_summary(self, cell_height: str = None) -> HTML:
```

Comment:
wouldn't it be best to start having this only with the new api?

Reply:
I agree, but I think we should first add the remaining visualizations in the new API, then change the viz examples, and then start implementing new visualization just on the new format. Since this is being done before all these changes, I'll keep it in the old format, and start that work of migrating to the new API/refactor examples in a future PR.